.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+### Description
+<!--
+Include issue references (e.g., fixes [#issue](link))
+include check boxs to describe the tesps to do or have been done
+Include necessary implementation details, including clarifications, disclaimers etc, related to the approach used(e.g. I opted to use this algorithm because ... and test it in this way ...).
+-->
+
+### Changelog
+<!--
+Include changelog entry (e.g. Fixed an unexpected change in recenter button when resuming the app.).
+See https://github.com/mapbox/navigation-sdks/blob/main/documentation/android-changelog-guidelines.md.
+You can remove the changelog block and add a `skip changelog` label, when applicable.
+ -->
+ <changelog></changelog>
```

Comment:
We maintain a running changelog in CHANGELOG.md, so there can be a reminder to update that file, but we shouldn’t encourage contributors to add a changelog entry here because it’ll get lost in the mix.

Reply:
I see. Thanks for the suggestion! I'll remove this part!